src/writer/vote.ts

```diff
@@ -51,7 +51,6 @@ export async function verify(body): Promise<any> {
       msg.space,
       jsonParse(proposal.strategies),
       proposal.network,
-      snapshot.utils.getProvider(proposal.network),
```

Comment:
We still need this change, which is not done in https://github.com/snapshot-labs/snapshot-hub/pull/147